client/src/theme/styles/components/global/annotation/_meta.scss

```diff
@@ -72,5 +72,9 @@
     @include respond($break50) {
       padding: 0;
     }
+
+    .overflow-wrapper {
+      width: 100%;
+    }
```

Comment:
One note also about the selector here. Compound selectors like this (`.annotations-meta .overflow-wrapper`) are generally a no-no, because this component could be the child of another component that also uses an `.overflow-wrapper` class (like `.annotations-list .overflow-wrapper`), and then there's a conflict. That's one of the core CSS issues on Manifold. There are styles for classes like `.input` that are then redefined within in lots of other classes, like `.form-secondary`. It makes it really difficult to refactor styling, because not only are styles for a particular DOM instance coming from two or three different places in code with conflicting styles, but the higher specificity of the compound selector means it can only be overridden with a selector of even higher specificity. We introduced BEM to help solve that but it takes a lot of discipline to keep up. Since BEM selectors are a single class and never supposed to be nested, a codebase that uses it will have a very flat graph of selectors without any competition between them. It makes everything a lot more predictable for devs to maintain. I'd say Manifold is about 50% on this approach now, but much of the core of it is not, and it's a pain.

OK sorry for the rant. What you did was not a big deal. I just tend to flag it whenever I see it on projects because I do think on a large scale it introduces a lot of technical debt. And it's an issue regardless of whether we're using Sass or CSS-in-JS.

Reply:
You weren't missing anything! Just me adding unnecessary widths as I'm wont to do...

Reply:
The only thing that was actually needed was the additional `div` so the padding is maintained when the text gets truncated.